packages/liveblocks-node/src/Session.ts

```diff
@@ -43,7 +43,7 @@ const READ_ACCESS = Object.freeze([
  */
 const FULL_ACCESS = Object.freeze(["room:write", "comments:write"] as const);
 
-const roomPatternRegex = /^[^*]{1,128}[*]?$/;
+const roomPatternRegex = /^(?=.+)[^*]{0,128}[*]?$/;
```

Comment:
When possible, I'd try to avoid lookaround patterns in regexes. In some cases they are unavoidable, but in this case, it looks like you're only using it to enforce that the regex isn't the empty string, correct?

Maybe simpler to read and understand if you just add the `*` case explicitly?

```suggestion
const roomPatternRegex = /^([*]|[^*]{1,128}[*]?)$/;
```

Reply:
yes!